luxtronik/scripts/dump_changes.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python3
+
+# pylint: disable=invalid-name, disable=too-many-locals
+
+"""Script to dump all value changes from Luxtronik controller"""
+
+import os
+import argparse
+
+from luxtronik import Luxtronik
+
+
+def dump_changes():
+    """Dump all value changes from Luxtronik controller"""
+    # pylint: disable=duplicate-code
+    parser = argparse.ArgumentParser(
+        description="Dumps all value changes from Luxtronik controller"
+    )
+    parser.add_argument("ip", help="IP address of Luxtronik controller to connect to")
+    parser.add_argument(
+        "port",
+        nargs="?",
+        type=int,
+        default=8889,
+        help="Port to use to connect to Luxtronik controller",
+    )
+    args = parser.parse_args()
+
+    client = Luxtronik(args.ip, args.port)
+    prev_calcs, prev_params, prev_visis = client.read()
+    # pylint: enable=duplicate-code
+    changes = {}
+
+    while True:
```

Comment:
This while loop does not have any `sleep()` in it; i.e., as soon as the data is compared to the old data, new data is queried from the heat pump. This is not sensible, since (IMHO) the heat pump updates its data once every second.

Reply:
I totally agree! 
This was not authored by me originally so I didn't look at it closely. I'll add a sleep.

Reply:
Hey, you introduced a magic constant ;)

From my perspective, there are some other issues with this `dump_changes` (disclaimer: I only read the source and never tried it). However, I suggest to finish this PR and deal with `dump_changes` afterwards.